workshop-cli/WorkshopCli/GitHubManager.cs

```diff
@@ -20,61 +21,88 @@ public void CreateBranch()
             UseShellExecute = false,
             CreateNoWindow = true
         };
-
-        var process = new Process { StartInfo = processStartInfo };
-        process.Start();
-        process.WaitForExit();
+        //Console.WriteLine("3");
+        try
+        {
+            var process = new Process { StartInfo = processStartInfo };
+            process.Start();
+            process.WaitForExit();
+        }
+        catch (Exception ex)
+        {
+            //Console.WriteLine($"Error executing Python script: {ex.Message}");
+        }
```

Comment:
Este try catch está a forçar ignorar exceções vindas de processos que estejas a correr. Não vais saber se alguma coisa não estiver a funcionar. O Console.WriteLine tava bem, faz sentido estar aí e resolver os problemas que estejam a aparecer.